### ISSUE TITLE
Unable to reproduce results when training densenet from scratch

### ISSUE THREAD

**Issue by praveen5733** [facebookresearch/odin#13]
I am able to reproduce the results reported in the paper when I use the pretrained models provided in the repo. 
But when I train a densenet from scratch the results are poorer compared to the report. Did anyone face a similar problem?

**Comment by tangbohu** [comment]
Also do I. Have you solved it?

**Comment by YixuanLi** [comment]
We are unable to update the github repo at this moment. However, we have recently built another repo which provides ODIN as well as many other OOD detection methods. Can you try this: https://github.com/jfc43/informative-outlier-mining?

**Comment by huberl** [comment]
I face the same problem @praveen5733  @tangbohu

**Comment by YixuanLi** [comment]
Can you provide details on your experimental results, so that I can take a
look at the difference? It's possible that the performance will have some
variations across model runs.

**Comment by Jiejiegary** [comment]
Hi, all, I have the similar problem. I trained densenet (and wideresnet) on cifar10 where models have normal test accuracy. When I test the model with odin in this task, I saw a pretty huge gap between the results and the reported ones. Maybe I miss something here.

**Comment by YixuanLi** [comment]
For wideresnet, you can refer to our latest paper: https://github.com/wetliu/energy_ood. It's also recommended to use energy score as it's parameter-free and gives a performance that's comparable or better than ODIN. 

For ODIN, you can typically get a ballpark performance estimation by setting the temperature to be T=1000.
